app/index.html

```diff
@@ -1,7 +1,20 @@
-<html>
-  <link rel="stylesheet" type="text/css" href="dist.css">
-  <title>Simple Markdown Editor</title>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="katex/katex.min.css" />
+    <link rel="stylesheet" type="text/css" href="dist.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <title>Markdown Basic Editor</title>
+  </head>
+
   <body>
-    <script type="text/javascript" src="dist.js"></script>
+    <!-- Dependencies -->
+    <script src="react/umd/react.production.min.js"></script>
```

Comment:
All dependencies should be compiled into dist.js.

Reply:
I'm confused about this. Both the TypeScript docs and React say to load React separately: 
- https://www.typescriptlang.org/docs/handbook/react-&-webpack.html
- https://reactjs.org/docs/optimizing-performance.html#single-file-builds

Reply:
I think making React and React-DOM load outside is supposed to make the time required for the initial load

Reply:
I think loading React separately is like having a `dist.css` instead of having the css in `dist.js`

Reply:
We use a bundler (webpack) which compiles everything into one file. So we don't need to follow their approach.

Reply:
React and React-DOM are now compiled into `dist.js` 👍